star-ui/src/components/FirmwarePanel.tsx

```
@@ -0,0 +1,118 @@
+                <div style={{ display: 'flex', gap: '8px' }}>
+                    <button
+                        onClick={simulateUpload}
+                        disabled={stage === 'uploading'}
+                        style={{
+                            flex: 1, padding: '8px', border: 'none', borderRadius: '6px',
+                            background: COLORS.primary, color: '#fff',
+                            fontSize: '11px', fontWeight: 600, cursor: 'pointer',
+                            opacity: stage === 'uploading' ? 0.5 : 1,
+                            letterSpacing: '0.06em', textTransform: 'uppercase',
+                        }}
+                    >
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)